packages/bootstrap-4/src/DescriptionField/DescriptionField.tsx

```diff
@@ -1,24 +1,22 @@
-import React from 'react';
+import React from "react";
 
-import { FieldProps } from '@rjsf/core';
+interface DescriptionFieldI  {
+    description? : string
+}
 
-import { makeStyles } from '@material-ui/styles';
-import Typography from '@material-ui/core/Typography';
-
-const useStyles = makeStyles({
-  root: {
-    marginTop: 5,
-  },
-});
-
-const DescriptionField = ({ description }: FieldProps) => {
+const DescriptionField = ({ description }: DescriptionFieldI) => {
   if (description) {
-    const classes = useStyles();
-
     return (
-      <Typography variant="subtitle2" className={classes.root}>
-        {description}
-      </Typography>
+      <>
+        <style type="text/css">
```

Comment:
@anikethsaha could you explain to me why we want to render an inline style tag here? Seems like this could potentially have interesting side effects that break styling across the site after the component mounts?

Reply:
It seems it is the recommended way to style in `react-bootstrap` ? https://react-bootstrap.github.io/getting-started/theming/#custom-styles-variants

I can switch it to `style` attribute if its ok

Reply:
If I understand the linked documentation section correctly, that's for generating new bootstrap css classname variants.

In their example:
```jsx
<>
  <style type="text/css">
    {`
    .btn-flat {
      background-color: purple;
      color: white;
    }

    .btn-xxl {
      padding: 1rem 1.5rem;
      font-size: 1.5rem;
    }
    `}
  </style>

  <Button variant="flat" size="xxl">
    flat button
  </Button>
</>
```

I think the `variant="flat"` prop will result in the element getting a CSS className `.btn-flat` (presumably because the Bootstrap base class name prefix for buttons is `btn`, and the `<style>` element is rendered to match all `.btn-flat` elements on the page.

That makes sense (and it's quite clever, though feels a bit dirty to be littering the page with multiple duplicate `style` tags for each instance of Button), but the style rule will be applied globally. This means that doing a `h2` selector rule here will apply a margin to all `h2` elements on the page.

I'm not crazy about using a `style` prop directly on the element either, because it then becomes impossible to override from user's own CSS without using `!important`.

I'll do a bit of research and get back to you.

Reply:
@anikethsaha I had a quick look at Bootstrap 4 [Spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/#how-it-works) documentation, and I think we should us the `mt-{size}` utility class to set margin-top relative to the user's own spacing sizes, e.g.
```jsx
  <h2 className="mt-2">{description}</h2>
```

You can play with the values between `mt-0` and `mt-5` to get the sizing right.

Reply:
Ohh cool, I will try that.

Reply:
done 👍